plot_data/templates.py

```diff
@@ -39,7 +39,8 @@
         <button name="Zoom-" value="OK" type="button" onclick="plot_data.zoomOut()"> Zoom- </button> &nbsp;&nbsp;
         Cluster:&nbsp; <input type="range" class="slider" min="0" max="2500" value="1250" onclick="plot_data.simpleCluster(value / 10000)"></input>
         <button name="resetClusters" value="OK" type="button" onclick="plot_data.resetClusters()"> Reset clusters </button> &nbsp;&nbsp;
-        <button name="resetView" value="OK" type="button" onclick="plot_data.resetView()"> Reset view </button>
+        <button name="resetView" value="OK" type="button" onclick="plot_data.resetView()"> Reset view </button> &nbsp;&nbsp;
+        <button name="showPoints" value="OK" type="button" onclick="plot_data.switchShowPoints()"> Show points </button>
```

Comment:
A neat way to say "switchShow" is "toggle", you could go for `togglePoints`, here

![image](https://github.com/Dessia-tech/plot_data/assets/42537234/7e754b7b-895b-473a-8b06-9add510e083d)

Reply:
Replaced